tools/run_tests/run_tests.py

```diff
@@ -490,8 +486,7 @@ def build_steps(self):
         # building for electron requires a patch version
         self.node_version += '.0'
       return [['tools/run_tests/helper_scripts/{}.sh'.format(build_script),
-               self.node_version,
```

Comment:
nit: I think this change also needs to be made on lines 479 and 480 above, in the windows case.